api/server.go

```diff
@@ -58,13 +58,12 @@ type Entry struct {
 func NewApiServer(cfg *ApiConfig, settings map[string]interface{}, backend *storage.RedisClient) *ApiServer {
 	rpcDaemons := [common.HierarchyDepth]*rpc.RPCClient{}
 
-	for level := 0; level < common.HierarchyDepth; level++ {
-		rpcConfig := settings["Upstream"].([]interface{})[level].(map[string]interface{})
-		rpcDaemons[level] = rpc.NewRPCClient(
-			rpcConfig["Name"].(string),
-			rpcConfig["Url"].(string),
-			rpcConfig["Timeout"].(string))
-	}
+	rpcConfig := settings["Upstream"].([]interface{})[0].(map[string]interface{})
+	rpcDaemons[0] = rpc.NewRPCClient(
```

Comment:
What is magic number 0 here? Can we make it non-magical? I strongly prefer muggle numbers

Reply:
I am going to steal 'muggle' numbers for myself